<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
color: black;	
border: 0px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="cred_box">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-color: #6498BB;
	border: 20px;
	border-radius: 10px;
}

QLineEdit:hover {
	background-color: #73A2C1;
}

QPushButton {
	background-color: #9ABE74;
}

QPushButton:hover{
	background-color: #A4C482;
}

QPushButton:pressed {
	background-color: #86B37E;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QGroupBox" name="ejournal">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QGroupBox" name="username_ej_box">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="username_ej_lable">
              <property name="text">
               <string>Имя пользователя от elshcool.ru</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="username_ej">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>50</height>
               </size>
              </property>
              <property name="inputMask">
               <string notr="true"/>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="maxLength">
               <number>32767</number>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="password_ej_box">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="password_lable">
              <property name="text">
               <string>Пароль от elshcool.ru</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="password_ej">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>50</height>
               </size>
              </property>
              <property name="inputMethodHints">
               <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
              </property>
              <property name="inputMask">
               <string notr="true"/>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="register_btn">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 0px;
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
